src/components/HeatMap/Row/Row.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Cell from '../Cell';
+import { Wrapper, RowName, Text } from './Row.styles';
+
+
+export default function Row({
+  data, dayLabel, row, getActiveCell, activeCell,
+}) {
+  return (
+    <Wrapper>
+      <RowName>
+        <Text>
+          {dayLabel}
+        </Text>
+      </RowName>
+      {
+        data.map((item, i) => (
```

Comment:
`i` seems to be the column. So maybe rename it to `column`. Or even better `hour`, because I guess it's the hour, right?